src/Card/Card.component.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from "react"
+import styled, { css } from "styled-components"
+import CardBody, { CardBodyProps } from "./CardBody.component"
+import CardFooter, { CardFooterProps } from "./CardFooter.component"
+import CardHeader, { CardHeaderProps } from "./CardHeader.component"
+import CardImage, { ImageProps } from "./CardImage.component"
+import CardFlipper, { CardFlipperProps } from "./CardFlipper.component"
+
+interface CardComposition {
+  Body: React.FC<CardBodyProps>
+  Header: React.FC<CardHeaderProps>
+  Footer: React.FC<CardFooterProps>
+  Image: React.FC<ImageProps>
+  Flipper: React.FC<CardFlipperProps>
+}
+
+export interface CardProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * Card will expand across available width
+   * @default false
+   */
+  fluid?: boolean
+  /**
+   * Raises the card, applying a small box-shadow
+   * @default false
+   */
+  raised?: boolean
+  /**
+   * Main image to be displayed
+   *
+   */
+  image?: string
+  /**
+   * Translate Y on hover
+   * @default false
+   */
+  hoverable?: boolean
+}
+
+const CardContainer = styled.div<CardProps>`
+  display: flex;
+  flex-direction: column;
+  border: 1px solid rgba(0, 0, 0, 0.1);
+  background: #fff;
+  border-radius: 5px;
+  overflow: hidden;
+  box-shadow: 2px 2px 5px rgba(0, 0, 0, 0.1);

```

Comment:
Styles look good! Once we add a `theme` object, we definitely want to make sure we go back to each component using literal style values like this one and use `style-system` props instead. This will help the components be more consistent and dynamically update when/if the theme changes!